ai-service/src/ai_service/main.py

```
@@ -1,27 +1,46 @@
-
 from ai_service.ollama_client import chat_with_ollama
-
 from .exceptions import AIServiceError
 
 
+def upload_code(code: str):
+    embedding = embed_text(code)
+    db.add_chunks([code], [embedding])
+
+
+def answer_question(user_question: str, top_k: int = 3):
```

Ruff (0.12.2)

Missing return type annotation for public function `answer_question`

Add return type annotation: `None`

(ANN201)